src/statefulmodelinstance.cpp

```diff
@@ -19,16 +19,20 @@ using namespace InferenceEngine;
 
 namespace ovms {
 
-uint64_t extractSequenceId(const tensorflow::TensorProto& proto) {
-    if (proto.uint64_val_size() == 1)
-        return proto.uint64_val(0);
-    return 0;
+const Status extractSequenceId(const tensorflow::TensorProto& proto, uint64_t* sequence_id) {
```

Comment:
Can we use reference instead of a pointer in `sequence_id`? 
Also please use camel case - `sequenceId`